### DISCUSSION TITLE
"Speak readings" failing consistently

### DISCUSSION THREAD

**Discussion by machiyag** [NightscoutFoundation/xDrip#2326] [Q&A]
What is going on? I used to have "speak readings" checked for much of the day, and several weeks ago it just stopped speaking. Since I did not always have it on, it took me some time to realize that I had a real problem. Because I was not getting audible updates, I have gotten myself in trouble. I've been working outside moving wheelbarrow-loads of wood chips, wearing work gloves, and unable to easily check my phone. Would have been helpful to know how low my bg was.

![xDrip log](https://user-images.githubusercontent.com/43839319/189506827-9109f277-68a5-44f4-991e-b9ff7a4daced.png)
The screen shot might not contain anything useful, but it's all I have to offer.

As far as I know, nothing has changed in my settings. Any thoughts and suggestions will be appreciated.

Many thanks,

mh

**Comment by jamorham** [comment]
I think this implies the text to speech feature of the phone is not working. First off I would try a reboot and if that doesn't work then the next steps would be to test the text to speech via another app and make sure the voices are all downloaded correctly etc. Other than that if the phone had been updated and there was some incompatibility with a newer android version then we could investigate that but this is the first report that I'm aware of so I would go for reboot first.

**Comment by machiyag** [comment]
Hmmm... I shut the detested Google Assistant off, and that might be the
cause.

mh

**Comment by machiyag** [comment]
I turned Assistant back on, and nothing. The phone, when I went into
Settings and clicked to test text to speech, happily said "this is an
example of text in English." Still no spoken readings.

After an additional hard restart, it just spoke. Thank you. Now if I try to
get the nosy Assistant out of my life, I'll do it one service at a time,
and make notes.

mh
